apps/main-cf.yml

```
@@ -167,6 +167,17 @@ Resources:
           - Status: Enabled
             AbortIncompleteMultipartUpload:
               DaysAfterInitiation: 1
+      CorsConfiguration:
+        CorsRules:
+          - AllowedHeaders:
+              - '*'
+            AllowedMethods:
+              - GET
+              - HEAD
+            AllowedOrigins:
+              - null
+            ExposeHeaders: []
+            MaxAge: 3000
```

Per https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-s3-bucket-cors-corsrule.html , only AllowedMethods and AllowedOrigins are required, so I'd err on the side of not specifying the rest unless we have a specific reason:
```suggestion
            AllowedMethods:
              - GET
              - HEAD
            AllowedOrigins:
              - "*"
```

---

per brians advice the other 2 are required

---

`AllowedOrigins` also specifically has to be `null`